kubernetes/config/scheduler.yaml

```diff
@@ -1,7 +1,20 @@
 apiVersion: v1
 data:
-  scheduler.json: |
-    {}
+  scheduler.jsonnet: |
+    local common = import 'common.libsonnet';
+
+    {
+      httpListenAddress: common.httpListenAddress,
+      clientGrpcServers: [{
+        listenAddresses: [':8982'],
+        authenticationPolicy: { allow: {} },
+      }],
+      workerGrpcServers: [{
+        listenAddresses: [':8983'],
+        authenticationPolicy: { allow: {} },
+      }],
+      blobstore: common.blobstore,
```

Comment:
If I'm not mistaken this is no longer valid after [this commit](https://github.com/buildbarn/bb-remote-execution/commit/e9cc231ea6359361f498b6a539b32e5f2c853634). Looks like you can just rename it and add `.contentAddressableStorage` on the end as seen [here](https://github.com/buildbarn/bb-deployments/blob/81abe8f7c899d43b1061bd70fc2c64a915d2937f/docker-compose/config/scheduler.jsonnet#L14).

Reply:
Ah yes, once I update the images this won't work. But I'd argue that is a job for https://github.com/buildbarn/bb-deployments/pull/18 :smile: